e2e/test/scenarios/embedding-sdk/static-dashboard.cy.spec.js

```diff
@@ -23,6 +23,9 @@ describeSDK("scenarios > embedding-sdk > static-dashboard", () => {
     cy.signInAsAdmin();
     setTokenFeatures("all");
     setupJwt();
+    cy.request("PUT", "/api/setting", {
```

Comment:
This is needed because all tests will be running with CORS, now, the one thing we keep here is to let storybook sign JWT because we don't use custom domain in this suite.